app/boards/shields/jorne/jorne.conf

```diff
@@ -0,0 +1,6 @@
+# Uncomment the following lines to enable the Jorne RGB Underglow
+# ZMK_RGB_UNDERGLOW=y
```

Comment:
```suggestion
# CONFIG_ZMK_RGB_UNDERGLOW=y
```

Reply:
Copied that from the `crkbd` config. So that's why the LEDs weren't working!

Reply:
It looks like this change was missed @krikun98.